projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -208,6 +209,11 @@ jQuery( document ).ready( function ( $ ) {
 				$( '#jitm-banner__activate a' ).text( window.jitm_config.activated_module_text );
 				$( '#jitm-banner__activate a' ).attr( 'disabled', true );
 
+				// Redirect to module settings page if a redirect is set in the JITM envelope.
+				if ( $activate_button.data( 'redirect' ) ) {
+					window.location.href = window.jitm_config.jetpack_admin_url + $activate_button.data( 'redirect' );
+				}
+
```

Comment:
Something about this doesn't sit right.

From a UX perspective, it seems unexpected that clicking an "Activate" button would redirect the user IMO.

I also think redirecting with `window.location.href` by pulling from the `window` object and the DOM might come with some risks. It could leave the door open to redirect somewhere else if another script from a different plugin decided to do something unfriendly and manipulate the window and DOM. 

cc @IanRamosC for some additional thoughts/ second opinion on this redirect method.

Would it work to swap the activate button for a "settings" button after the activation is complete?

Reply:
> Would it work to swap the activate button for a "settings" button after the activation is complete?

Interesting - the main issue was the fade away without a clear next step. Swapping it to stay in place and have Activate => Settings would work. Then they can dismiss it with the "X" or click settings to go to where the re-direct would be. 

Let me modify on that basis.

Reply:
>cc @IanRamosC for some additional thoughts/ second opinion on this redirect method.

Just saw the comment now (that this seems to be resolved already), but I wanted to leave a comment for posterity. I agree with you. It's risky to redirect that way, especially if the redirect URL comes from the `window` object or anything else we don't have control over unless we validate it.